### ISSUE TITLE
Integrate with Apollo Data

### ISSUE THREAD

**Issue by orlaqp** [derickbailey/mustbe#17]
First of all, thanks for all your work/sharing .. I have been following you for some time now. I think the idea behind `mustBe` its pretty cool and I would like to make it even more open. For example in my case i would like to call the authorization method from inside `graphql resolvers`. So no really a route because all graphql queries and mutations run form a single endpoint and also not really a middleware because I do not have enough data in the middleware to make a decision about authorization. 

Basically what I would like to do is to call these methods from anywhere in my app. Do you think this would be a big change?
